src/web/components/ApiKeyManagement/ApiRolesCell.scss

```diff
@@ -0,0 +1,12 @@
+.api-roles-cell {
+  display: flex;
+  font-size: 0.75rem;
+  line-height: 0.75rem;
```

Comment:
We prefer to use `line-height` in a unitless manner to act as a multiplier. It avoids situations where the text is too big for the line-height. You may need to change the number from 0.75 though

Reply:
That makes sense. Should I change the same thing in `ParticipantManagementItem.scss` since that has `.participant-item-types` which uses the same css.

Reply:
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/152250444/b987f341-72b6-4841-afbc-bf0e2ae657de)
Have done this for both. Used 1.15 to match the line height for buttons which is being used by the apiRolesCell when showing the tooltip